src/components/store/createStore.ts

```diff
@@ -0,0 +1,56 @@
+import { EditorState } from './types/EditorState';
+import { Reducer } from './types/Reducer';
+import { Action } from './types/Action';
+import { Store } from './types/Store';
+
+/**
+ * Function creates store and returns functions for use it
+ *
+ * @param reducer - current Editor reducer function
+ * @param initialState - initial state of the store
+ */
+const createStore = (reducer: Reducer, initialState: EditorState = {}): Store => {
```

Comment:
why anonymous function? It would better to use named function insted, i think.

```suggestion
function createStore(reducer: Reducer, initialState: EditorState = {}): Store => {
```
```